### ISSUE TITLE
Easy_install not working

### ISSUE THREAD

**Issue by PratibhaVL** [felipecode/coiltraine#20]

**Comment by chandanavgermany** [comment]
Did you find the solution?

**Comment by PratibhaVL** [comment]
yes. I have a question about the 24 GB dataset provided, does it follow the traffic lights rule?


Regards,

Pratibha Laturiya,

Graduate student,

Colorado State University,

+19708259039

________________________________

**Comment by chandanavgermany** [comment]
@PratibhaVL  I don't know, I am still facing issue regarding running the program.
